src/models/bracket_races.rs

```diff
@@ -125,11 +135,19 @@ impl BracketRace {
         &self,
         conn: &mut SqliteConnection,
     ) -> Result<(Player, Player), diesel::result::Error> {
-        // TODO multiple queries bad 😫
-        let p1 =
-            Player::get_by_id(self.player_1_id, conn)?.ok_or(diesel::result::Error::NotFound)?;
-        let p2 =
-            Player::get_by_id(self.player_2_id, conn)?.ok_or(diesel::result::Error::NotFound)?;
+        let mut players = crate::schema::players::table
```

Comment:
this is a driveby and query optimization isn't _really_ important but it's annoyed me for ages that this is 2 dumb queries